weave-js/src/errors.ts

```diff
@@ -0,0 +1,57 @@
+import {WeaveApp} from './weave';
+
+export class UseNodeValueServerExecutionError extends Error {}
+
+export const errorToPayload = (
+  error: Error,
+  weave?: WeaveApp
+): {[prop: string]: any} => {
```

Comment:
Can change return type be `Record<string, any>` to be more readable?

Reply:
done